Makefile.tools.mk

```diff
@@ -30,7 +30,19 @@ POETRY_VERSION ?= 1.8.3
 .PHONY: golangci-lint
 golangci-lint: $(GOLANGCI_LINT)
 $(GOLANGCI_LINT): $(LOCALBIN)
-	$(call go-install-tool,$(GOLANGCI_LINT),github.com/golangci/golangci-lint/cmd/golangci-lint,$(GOLANGCI_LINT_VERSION))
+	@if [ -x "$(GOLANGCI_LINT)" ]; then \
```

Comment:
I've encountered issues here, similar to:
```
The Go language version (go1.23) used to build golangci-lint is lower than the targeted Go version (1.24)
```

However, it was a local setup problem due to multiple versions of GoLang itself and golangci-lint libraries available in the system.
To fix it locally, I was to make sure that:
- the `golangci-lint` binary available in the GOPATH was the desired version and;
- The GoLang version env is properly set to 1.24.

In my case, I had a problem where the GoLand IDE had installed Go 1.24 by itself, and the GOROOT was pointing to another location. Once I fixed it together, ensuring there was no other lint library, the issue was gone.